recipe/conda_build_config.yaml

```diff
@@ -1,4 +1,13 @@
 c_compiler_version:        # [linux]
-  - 11                     # [linux]
+  - 11                   # [linux]
 cxx_compiler_version:      # [linux]
-  - 11                     # [linux]
+  - 11                   # [linux]
+cxx_compiler:              # [win]
+  - vs2019                 # [win]
+c_compiler:                # [win]
+  - vs2019                 # [win]
```

Comment:
We might want to try with vs2017 to see if it fixes the compilation errors.

Reply:
It didn't build with `vs2017` because of this error:
`C:\b\abs_f4ohu1gj3y\croot\grpc-suite_1697462683394\_h_env\Library\include\absl/base/policy_checks.h(50): fatal error C1189: #error:  "This package requires Visual Studio 2019 (MSVC++ 16.0) or higher."`